src/apps/service/apps/job_listings.clj

```diff
@@ -45,6 +50,7 @@
     :app_name        (:app-name job)
     :description     (:description job)
     :enddate         (job-timestamp (:end-date job))
+    :system_id       (string/lower-case (:job-type job))
```

Comment:
Will the system ID always be the lowercased job type? If so it seems like a lot of code could be condensed/made dependent on this fact. Minor, but might be worth considering if we're enforcing this.